packages/core/rendering/core-rendering-server-internal/src/views/template.tsx

```diff
@@ -45,7 +45,7 @@ export const Template: FunctionComponent<Props> = ({
         <meta charSet="utf-8" />
         <meta httpEquiv="X-UA-Compatible" content="IE=edge,chrome=1" />
         <meta name="viewport" content="width=device-width" />
-        <title>{title}</title>
+        <title>{title}🧔🏻‍</title>
```

Comment:
NIT: I would add a space between the title and @spalger

Reply:
Nit: Replace 🧔🏻‍♂️ with ![image](https://user-images.githubusercontent.com/1178348/217867285-b23dcf1f-1a4a-45fd-b828-f6b24215fef2.png)